gh_oidc_role/examples/oidc_add_assume/main.tf

```diff
@@ -0,0 +1,39 @@
+locals {
+  readonly_role = "GHOIDCReadOnlyRole"
+}
+
+module "gh_oidc_roles" {
+  source            = "../../"
+  billing_tag_value = "terratest"
+  roles = [
+    {
+      name      = local.readonly_role
+      repo_name = "terraform-modules"
+      claim     = "ref:refs/heads/main"
+    }
+  ]
+  policies = data.aws_iam_policy_document.service_principal.json
+}
+
+data "aws_iam_policy" "admin" {
```

Comment:
Super minor, but could you change the resource name to something like:
```hcl
data "aws_iam_policy" "read_only"
```

Reply:
It would make more sense so +1 to that!

Reply:
good catch, updated!